azurerm/resource_arm_role_assignment_test.go

```diff
@@ -435,3 +460,22 @@ resource "azurerm_role_assignment" "test" {
 }
 `, rInt, roleAssignmentID)
 }
+
+func testAccAzureRMRoleAssignment_managementGroupConfig() string {
+	return `
+data "azurerm_subscription" "primary" {}
+
+data "azurerm_client_config" "test" {}
+
+data "azurerm_builtin_role_definition" "test" {
+	name = "Monitoring Reader"
+}
+
+data "azurerm_management_group" "test" {}
+	resource "azurerm_role_assignment" "test" {
```

Comment:
Coould we fix this formatting:
```suggestion

resource "azurerm_role_assignment" "test" {
```

Reply:
Done.